rundeckapp/build.gradle

```diff
@@ -29,6 +29,13 @@ apply plugin:"com.energizedwork.idea-project-components"
 apply plugin:"asset-pipeline"
 apply plugin:"org.grails.grails-gsp"
 
+dependencyManagement {
+    imports {
+        mavenBom "org.grails:grails-bom:$grailsVersion"
+        mavenBom 'org.springframework.cloud:spring-cloud-starter-parent:Edgware.SR6'
```

Comment:
Curious that you had to add the grails and spring-cloud boms to the build. This shouldn't be necessary to include spring dependencies. did you run with some issues?

Reply:
I had to add this when I had my dev environment configured with the new enterprise reloadable config plugin. The way our gradle project is set up caused gradle to be unable to find the dependencies until I added this section on the OSS side. We could probably leave it out for the OSS build, but if anyone tries to use the pro config plugin in their dev env it will fail if this section isn't present. Ideas?